modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -1347,6 +1347,34 @@ public void testCacheIdleVerify() throws Exception {
         assertContains(testOut.toString(), "conflict partitions");
     }
 
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testCacheIdleVerifyNodeFilter() throws Exception {
+        IgniteEx ignite = startGrids(3);
+
+        ignite.cluster().active(true);
+
+        UUID lastNodeId = ignite.localNode().id();
+
+        ignite.createCache(new CacheConfiguration<>(DEFAULT_CACHE_NAME)
+            .setAffinity(new RendezvousAffinityFunction(false, 32))
+            .setNodeFilter(node -> !node.id().equals(lastNodeId))
+            .setBackups(1));
+
+        try (IgniteDataStreamer streamer = ignite.dataStreamer(DEFAULT_CACHE_NAME)) {
+            for (int i = 0; i < 100; i++)
+                streamer.addData(i, i);
+        }
+
+        injectTestSystemOut();
+
+        assertEquals(EXIT_CODE_OK, execute("--cache", "idle_verify", DEFAULT_CACHE_NAME));
+
+        assertContains(testOut.toString(), "no conflicts have been found");
```

Comment:
Please add assert on correct number of found partitions.

Reply:
Without --dump number of found partitions is not printed so there's nothing to check.